### ISSUE TITLE
Checklist checkboxes can't be used with keyboard

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#213]
These checkboxes can't receive focus or be selected using a keyboard.

**Comment by Blackbaud-PatrickOFriel** [comment]
This is a problem with all of our fancy checkboxes that we'll have to address.

**Comment by Blackbaud-ToddRoberts** [comment]
I am not 100% sure it is an issue with the checkboxes, on at least some forms I checked you could tab to a checkbox but it would not look focused, so if you randomly tab and hit spacebar you could get the select/deselect but would have no way of knowing which field you were going to change ;)

**Comment by Blackbaud-PatrickOFriel** [comment]
Haha, well then we should definitely work on getting the visual focus indication issue sorted out first

**Comment by Blackbaud-PatrickOFriel** [comment]
Fixed by the fix for #215
